node/resolvers/Routes/index.ts

```diff
@@ -414,7 +414,10 @@ export const Routes = {
     phoneNumber = costCenterResponse?.data?.getCostCenterById?.phoneNumber
 
     businessDocument =
-      costCenterResponse.data.getCostCenterById.businessDocument
+      costCenterResponse.data.getCostCenterById.businessDocument?.replace(
```

Comment:
Are these non digit chars just hifens and dots, as in Brazil's CNPJ, for example? If so, I understand that we can just remove them, but this got me worried about use cases in other countries which might require non digit characters, so just wanted to understand better.